OpenRA.Mods.Common/Activities/HarvesterDockSequence.cs

```diff
@@ -41,7 +41,13 @@ public HarvesterDockSequence(Actor self, Actor refinery, int dockAngle, bool isD
 			DragOffset = dragOffset;
 			DragLength = dragLength;
 			Harv = self.Trait<Harvester>();
-			StartDrag = self.CenterPosition;
+
+			/*
```

Comment:
Code should be removed, not just commented out.  The commit message documents the change.  This seems like a separate bugfix, so should have its own commit.